### DISCUSSION TITLE
Question: Sold car, will get a new one. What will happen?

### DISCUSSION THREAD

**Discussion by ninoez** [teslamate-org/teslamate#1076] [Q&A] [answered]
Hi, does anyone know what happens if you replace your car? I sold my Model 3 and will get a new one in a few weeks. Will TeslaMate recongnize the new car or will it add the data to the old one? Will I still be able to access the old data? Am I able to archive the old data?

Thanks!

**Comment by kinteseay** [comment]
What ever happened?  I'm curious

**Reply by cwanja** [reply, marked as answer]
Two things should happen:
1. Your sold car will continue to collect data until the refresh token is expired by your account no longer having the vehicle. You could expedite this by logging of TeslaMate immediately after your vehicle is transferred to the new owner. 
2. Your new car will begin being collected upon refresh of your Tesla account. Again, you can expedite this by following the same process. Your new car will **only** begin logging once it has been accepted via Tesla. Thus there could be a gap in data from when you accept delivery and when you can get back to TeslaMate to cycle the login.
